tripy/nvtripy/flat_ir/ops/plugin.py

```diff
@@ -95,7 +96,10 @@ def plugin_field_to_attr(field_info: "compiler.PluginFieldInfo", values: Any) ->
         attrs = [ir.IntegerAttr.get(INT_TYPES[field_info.type](), value) for value in values]
 
     if len(attrs) != field_info.length:
-        return Result.err([f"Expected {field_info.length} value(s), but got {len(attrs)}."])
+        if field_info.length > 0:
+            return Result.err([f"Expected {field_info.length} value(s), but got {len(attrs)}."])
+        else:
+            logger.verbose(f"Expected {field_info.length} value(s), but got {len(attrs)}. May be ignored if using TensorRT quickly deployable plugins (QDP) or if it is variable length field.")
```

Comment:
What does `field_info.length == 0` mean in the normal case? Would a plugin ever have a list field that expects no entries? If not, we can probably just drop the warning here

Reply:
In that case the only change needed here is:
```py
if field_info.length > 0 and len(attrs) != field_info.length:
```

Reply:
It is a legal thing to do, although realistic use cases are hard to imagine. Might be better to just drop the warning if it is indeed rare.

Reply:
Dropped the warning